napari_allencell_annotator/controller/main_controller.py

```diff
@@ -73,20 +103,77 @@ def _csv_import_selected_evt(self, file_list: List[str]):
         file_list : List[str]
             The list containing one file name.
         """
-
+        self.already_annotated = None
         if file_list is None or len(file_list) < 1:
             self.images.view.alert("No selection provided")
         else:
-            # TODO
             file_path = file_list[0]
+            if os.path.splitext(file_path)[1] == ".json":
+                self.annots.read_json(file_path)
+
+            else:
```

Comment:
maybe make this explicitly an `if else` statement. We shouldn't be able to pass in a non csv/json to this point in the code but just in case :)